src/runtime/runtime-simd.cc

```diff
@@ -340,10 +376,29 @@ RUNTIME_FUNCTION(                               \
   CONVERT_ARG_CHECKED(SOURCE_TYPE, a, 0);                     \
                                                               \
   TARGET_TYPE::value_t result;                                \
-  for (int i = 0; i < SOURCE_TYPE::kLanes; i++) {             \
-    FUNCTION(a->getAt(i), &result.storage[i]);                \
+  if (#FUNCTION  == "BitsTo") {                               \
+    char *t1 = reinterpret_cast<char*>(&(a->get().storage[0]));\
+    char *t2 = reinterpret_cast<char*>(&result.storage[0]);   \
+    for (int i = 0; i < 16; i++) {                            \
+      FUNCTION(t1++, t2++);                                   \
+    }                                                         \
+  } else {                                                    \
+    if (SOURCE_TYPE::kLanes > TARGET_TYPE::kLanes) {          \
+      for (int i = 0; i < TARGET_TYPE::kLanes; i++) {         \
+        FUNCTION(a->getAt(i), &result.storage[i]);            \
+      }                                                       \
+    } else {                                                  \
+      for (int i = 0; i < SOURCE_TYPE::kLanes; i++) {         \
+        FUNCTION(a->getAt(i), &result.storage[i]);            \
```

Comment:
yes, I will reduce the iterations by getting the bytes of source type. And add a empty function to pass compilation.